diff --git a/node_modules/docdash/static/scripts/nav.js b/node_modules/docdash/static/scripts/nav.js
index 6dd8313..74d6131 100644
--- a/node_modules/docdash/static/scripts/nav.js
+++ b/node_modules/docdash/static/scripts/nav.js
@@ -2,6 +2,7 @@ function scrollToNavItem() {
     var path = window.location.href.split('/').pop().replace(/\.html/, '');
     document.querySelectorAll('nav a').forEach(function(link) {
       var href = link.attributes.href.value.replace(/\.html/, '');
+      link.href = link.attributes.href.value.replace(/\#\..+(?=\#)/, '')
       if (path === href) {
         link.scrollIntoView({block: 'center'});
         return;
